x/gamm/keeper/keeper.go

```diff
@@ -88,3 +88,9 @@ func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
 func (k Keeper) setParams(ctx sdk.Context, params types.Params) {
 	k.paramSpace.SetParamSet(ctx, &params)
 }
+
+// IsPermissionlessPoolCreationEnabled returns nil if permissionless pool creation in the module is enabled.
+// Error otherwise
+func (k Keeper) IsPermissionlessPoolCreationEnabled(ctx sdk.Context) error {
+	return nil
+}
```

Comment:
This is kind of odd imo. Any time I see an "Is" method I always expect a bool rather than an error.

Reply:
Renamed to `Validate`